src/odemis/driver/test/xt_client_test.py

```diff
@@ -995,6 +1009,34 @@ def test_beamlet_index_info(self):
         self.assertEqual(len(beamlet_index_info["range"]["x"]), 2)
         self.assertEqual(len(beamlet_index_info["range"]["y"]), 2)
 
+    def test_contrast(self):
+        """Test setting and getting the contrast."""
+        init_contrast = self.microscope.get_contrast()
+        new_contrast = init_contrast + 0.01
+        contrast_range = self.microscope.contrast_info()["range"]
+        if new_contrast < contrast_range[0]:
+            new_contrast = contrast_range[0] + 0.01
+        elif new_contrast > contrast_range[1]:
+            new_contrast = contrast_range[1] - 0.01
+        self.microscope.set_contrast(new_contrast)
+        self.assertAlmostEqual(new_contrast, self.microscope.get_contrast())
+        # Set dwell time back to initial value
+        self.microscope.set_contrast(init_contrast)
+
+    def test_brightness(self):
+        """Test setting and getting the brigthness."""
+        init_brightness = self.microscope.get_brightness()
+        new_brightness = init_brightness + 0.01
+        brightness_range = self.microscope.brightness_info()["range"]
+        if new_brightness < brightness_range[0]:
+            new_brightness = brightness_range[0] + 0.01
+        elif new_brightness > brightness_range[1]:
+            new_brightness = brightness_range[1] - 0.01
+        self.microscope.set_brightness(new_brightness)
+        self.assertAlmostEqual(new_brightness, self.microscope.get_brightness())
+        # Set dwell time back to initial value
+        self.microscope.set_brightness(init_brightness)
```

Comment:
I already told you via chat but otherwise it might look as if I don't want to review your PR:
Maybe also a test should also be included (on the real Hw) which tests if the brightness of acquired images changes when you change the contrast/brightness